tasks/section_01_level1.yml

```diff
@@ -22,7 +22,6 @@
     - jffs2
     - hfs
     - hfsplus
-    - udf
```

Comment:
@evaldofelipe It's wrong, you need fix that into goss-cis-benchmark project

Reply:
Thanks! Fixed on correct project.